README.md

```diff
@@ -22,36 +36,248 @@ component written in native code. Using that component from DNX
 currently [requires a workaround][grpc-workaround]; on Windows please run `fix_dnx_grpc.bat`
 before running any gRPC code.
 
-Documentation
--------------
+## Specifying a Project ID
```

Comment:
This section doesn't apply, and should be removed. We use an environment variable within the integration tests, but that's all. Anywhere else that you need a project ID, it has to be specified explicitly - at least at the moment.